com.github.artemanufrij.playmymusic.json

```diff
@@ -0,0 +1,155 @@
+                "sha256": "42fe6d651910cb8b720167f71c5255a1b7b1afc82fecd3f31e61f9602b3b1335"
+            }]
+        },
+        {
+            "name": "gst-libav",
+            "config-opts": [
+                "--disable-gtk-doc"
+            ],
+            "cleanup": ["*.la", "/share/gtk-doc"],
+            "sources": [{
+                "type": "archive",
+                "url": "https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.12.3.tar.xz",
+                "sha256": "015ef8cab6f7fb87c8fb42642486423eff3b6e6a6bccdcd6a189f436a3619650"
+            }]
+        },
+        {
+            "name": "gst-plugins-ugly",
+            "config-opts": ["--disable-gtk-doc"],
+            "cleanup": ["*.la", "/share/gtk-doc"],
+            "sources": [{
+                "type": "archive",
+                "url": "https://gstreamer.freedesktop.org/src/gst-plugins-ugly/gst-plugins-ugly-1.12.3.tar.xz",
+                "sha256": "e88ca584c94ea78eeecbf3af00ef7f134b66bdee7408aa4aa6c547235e060052"
+            }]
+        },
+        {
+            "name": "granite",
+            "buildsystem": "cmake",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_INSTALL_PREFIX=/app",
+                "-DCMAKE_INSTALL_LIBDIR=/app/lib"
+            ],
+            "build-options": {
+                "env": {
+                    "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_GIRDIR": "/app/share/gir-1.0",
+                    "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_TYPELIBDIR": "/app/lib/girepository-1.0"
+                }
+            },
+            "sources": [{
+                "type": "archive",
+                "url": "https://github.com/elementary/granite/archive/0.5.tar.gz",
+                "sha256": "cc4905ae70fddeba3d2ded44bb642be77d419aa090251a7ab24c155b8616be06"
+            }]
+        },
+        {
+            "name": "playmyvideos",
+            "buildsystem": "cmake",
+            "build-options": {
+                "config-opts": [
+                    "-DCMAKE_INSTALL_PREFIX=/app"
+                ],
+                "env": {
+                    "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_GIRDIR": "/app/share/gir-1.0",
+                    "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_TYPELIBDIR": "/app/lib/girepository-1.0"
+                }
+
+            },
+            "sources": [{
+                    "type": "archive",
+                    "url": "https://github.com/artemanufrij/playmymusic/archive/0.4.4.tar.gz",
+                    "sha256": "8325a2ded0a36235e028303ba76f4944ff803d4b4f936d5dd3a85804cf637c08"
+                },
+                {
+                    "type": "file",
+                    "path": "FindVala.cmake"
+                },
+                {
+                    "type": "file",
+                    "path": "ValaPrecompile.cmake"
+                },
+                {
+                    "type": "file",
+                    "path": "ValaVersion.cmake"
```

Comment:
Plan on fixing these missing cmake files upstream?

Reply:
Already reported that upstream. Hope they move to using meson instead ;)